crates/iceberg/src/avro/schema.rs

```diff
@@ -96,7 +98,13 @@ impl SchemaVisitor for SchemaToAvroSchema {
         _struct: &StructType,
```

Comment:
Hi, @liurenjie1024, why we need `StructType` here, it seems not been used?

Reply:
The `_struct` argument is from the trait `SchemaVisitor`.

Reply:
Undetstood. Thanks for the explanation.